webapp/components/Editor/Editor.spec.js

```diff
@@ -98,6 +111,16 @@ describe('Editor.vue', () => {
         expect(wrapper.vm.editor.extensions.options.hashtag.items()).toEqual(propsData.hashtags)
       })
 
+      it('limits the suggestion list to 15 users', () => {
```

Comment:
copy pasta! 🍝 😆 should be `hashtags` instead of `users` here...

Reply:
you caught me :open_mouth: